packages/api/cms-api/src/blocks/discover.service.ts

```diff
@@ -44,4 +70,29 @@ export class DiscoverService {
         });
         return ret;
     }
+
+    discoverTargetEntities(): DiscoverTargetEntitiesResult[] {
```

Comment:
this is currently in block module, but dependencies should be a block independent feature (in future).

Reply:
I extracted it to a `DependenciesModule`